region/tourian/main.json

```diff
@@ -4061,8 +4061,26 @@
               "id": 1,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Protected Shaft Climb",
                   "notable": false,
+                  "requires": [
+                    "ScrewAttack",
```

Comment:
charge works as well - pseudo screw

Reply:
pseudo doesn't seem intuitive here, so I would want a tech for it `canPseudoScrew`.